README.md

```diff
@@ -148,6 +148,13 @@ exports.LocalConfig = function (localizer, project) {
 
 ```
 
+## Choose a renderer
```

Comment:
I'd title "Custom renderer" to make clearer it's an optional step. What do you think?